incubator/hnc/internal/validators/setup.go

```diff
@@ -42,7 +50,7 @@ func CreateCertsIfNeeded(mgr ctrl.Manager, novalidation, internalCert bool) (cha
 		CaOrganization: caOrganization,
 		DNSName:        dnsName,
 		CertsMounted:   setupFinished,
-	}, vwhName)
+	}, rwh)
```

Comment:
This feels very strange to me. We're already dealing with one object - `CertRotator` - so why are we now dealing with _another_ object (`ReconcileWH`) that's really just an implementation detail of the rotator? I'd far rather only deal with one object here, and I think that should be the rotator, not the reconciler.

I'd go so far as to say that `ReconcileWH` should be private so it's not even accessible here.